client/src/components/sidebar.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react"
+
+export const SideBar = () => {
+  return (
+    <div className="fixed top-0 left-0 h-screen w-16 m-0 
+                    flex flex-col
+                    text-white shadow-lg bg-white">
+        <i></i>
```

Comment:
Why empty <i/>

Reply:
Problems with types when importing SVGs due to lack of tsconfig, so temporarily left empty for avatars and icons. Will complete this in future commits when types are set.